Header header
bool clearQueued
bool clearCurrent
---
Header header
string feedback